packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts

```diff
@@ -128,40 +121,59 @@ export const StyledButton = styled(Button)`
   width: 100%;
 `
 
-export const PaginationContainer = styled.div`
-  padding-top: ${sizes(6)};
+export const TabsWrapper = styled.div<{ isFiltersOpen: boolean }>`
+  z-index: 99;
```

Comment:
```suggestion
  z-index: ${zIndex.transactionBar};
```

Use this so it doesn't mess with the member dropdown

Reply:
@drillprop this need to be applied